internal/controller/ols_app_server_assets.go

```diff
@@ -259,6 +263,27 @@ func (r *OLSConfigReconciler) generateOLSConfigMap(ctx context.Context, cr *olsv
 			TranscriptsStorage:  "/app-root/ols-user-data/transcripts",
 		},
 		IntrospectionEnabled: cr.Spec.OLSConfig.IntrospectionEnabled,
+		QuotaHandlersConfig: QuotaHandlersConfig{
```

Comment:
If we make `QuotaHandlersConfig` really optinal [as suggested in this comment](https://github.com/openshift/lightspeed-operator/pull/716/files#r2075879272), we can set the value of `QuotaHandlersConfig` only when `cr.Spec.OLSConfig.QuotaHandlersConfig.EnableTokenHistory` is not empty. 
The generated config map will be shorter for administrator to inspect.